zip-0317.rst

```diff
@@ -245,19 +277,26 @@ Wallets SHOULD deploy these changes immediately. Nodes SHOULD deploy the
 change to the :math:`low\_fee\_penalty` threshold described in
 `Mempool size limiting`_ immediately.
 
-Nodes can deploy restrictions to their policies for relaying, mempool
-acceptance, and/or mining once a sufficient proportion of wallets in the
-ecosystem are observed to be paying at least the updated conventional
-transaction fee. Node developers SHOULD coordinate on deployment
-schedule.
+Miners can deploy restrictions to their policies for transaction inclusion,
+once a sufficient proportion of wallets in the ecosystem are observed to
+be paying at least the updated conventional transaction fee.
```

Comment:
Should this say "proportion of wallets" or "proportion of transactions"? The latter is more easily observable by looking at the blockchain.

Reply:
I'm +1 on "proportion of transactions".